modules/common/package.json

```diff
@@ -19,11 +19,13 @@
   },
   "license": "MIT",
   "peerDependencies": {
-    "@nestjs/common": ">=6.5",
-    "typescript": ">=3.5"
+    "@nestjs/common": "^6.11.11",
+    "@nestjs/core": "^6.11.11",
+    "typescript": ">=3.5",
+    "@bugsnag/js": "^6.5.2",
+    "@bugsnag/plugin-express": "^6.5.1"
```

Comment:
if you install deps in this repo, do they need to be listed as peer?

Reply:
I think so.. at least to my understanding of `peerDependencies`.. it might not like it if we have different versions of these packages installed wherever we are depending on nestjs-common